hatasmota/status_sensor.py

```diff
@@ -46,6 +48,7 @@
 #  "Module or Template":"Generic",             <NONE>
 #  "RestartReason":"Software/System restart",  stat/STATUS1:"StatusPRM"."RestartReason"
 #  "Uptime":"1T17:04:28",                      stat/STATUS11:"StatusSTS"."Uptime"; tele/STATE:"Uptime"
+#  "BatteryPercentage":60,                     Since Tasmota 13.0.0.3 optional
```

Comment:
These comments are meant to document where the various information is found.
For example, `Uptime` is present in the response to STATUS11 under key `Uptime` in the `StateusSTS` dict and in the tele/STATE under key `Uptime`. Can you improve the comment also for BatteryPercentage?